real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/localizers/HighTrustGPSLocalizer.java

```diff
@@ -39,51 +39,66 @@ public HighTrustGPSLocalizer(){
     	buggyFrameGpsX = 0.0;
     	buggyFrameGpsY = 0.0;
         buggyFrameRotZ = 0.0;
+        buggySteeringAngle = 0.0;// wheel direction in buggy frame
```

Comment:
IIRC low level and high level have definitions for the direction for steering.  If this hasn't been resolved, we should figure it out before summer is out